### ISSUE TITLE
[Feature Request] Option to disable mashup view and go straight to results.

### ISSUE THREAD

**Issue by xekon** [ampache/ampache#2881]
## Describe what you don't like about Ampache, or what it is missing

I do not use the mashup view, I was thinking of making it a separate page, currently you click Albums from the left which sends you to mashup, from there you can click into all albums, because of this extra click I just disable it. 

## Describe how you would like the feature to work**

I would prefer there to be an option to disable it and possibly to have it on a separate page.

currently I just do this after every ampache update:

```
	# Disable Mashup, go straight to results:
sed -i 's/mashup/browse/' /var/www/ampache/templates/sidebar_home.inc.php

	# Browse album_artist instead of artist (otherwise top NAV bar is missing):
sed -i 's/\?action=artist/\?action=album_artist/' /var/www/ampache/templates/sidebar_home.inc.php
```

**Comment by lachlan-00** [comment]
this one is also a develop / 5.0.0 change that's going in. browse will always be a browse now and i haven't decided how to add the mashup pages.
probably an optional show/hide mashup option with it in a separate section?

**Comment by usox** [comment]
Or maybe drop the mashup view completely and think about a new "discover" view which includes artists, albums, tags, ... - separated from the library browsing menu...?

**Comment by lachlan-00** [comment]
it's always been a weird one because you can just enable grid view yourself. the trending/recent/top part is really the only difference.

**Comment by xekon** [comment]
Should this be closed, seems 5.0 already skips the mashup...

**Comment by lachlan-00** [comment]
i do want to optionally put that back or create another section outside the information section
